lms/djangoapps/homepage/views.py

```diff
@@ -8,6 +8,9 @@
 
 
 def home_page(request, user=AnonymousUser()):
+    """
+    Override the default homepage for students. Display featured courses and communities
```

Comment:
I think this line should be updated too, like "Override default homepage for students" 

No need to tell what content this home page contains